populate_db.py

```diff
@@ -108,11 +127,75 @@ def populate_words(controller):
 
         print(message)
 
+def populate_db_with_info(host_name, host_type, controller_room, controller_trans):
+    guest_name = random.choice(guest_user_ids)
+
+    room_id = controller_room.generate_room_id()
+
+    result_room, message_room = controller_room.create_room(room_id, host_name, host_type)
+
+    if result_room == 1:
+        register_result, register_message = controller_room.register_user_in_room(room_id, guest_name)
+
+        print(f'RoomID: {room_id} GuestID: {guest_name}: {register_message}')
+
+        if register_result == 1:
+            guest_type = SPEAKER if host_type == SIGNER else SIGNER
+
+            host_info = [host_name, host_type]
+            guest_info = [guest_name, guest_type]
+
+            from_user_info = host_info
+            to_user_info =  guest_info
+
+            for _ in range(len(messages)):
+                message_to_send = random.choice(messages)
+
+                from_name = from_user_info[0]
+                to_name = to_user_info[0]
+                message_type = from_user_info[1]
+
+                result, create_message = controller_trans.create_message_entry(room_id, from_name, to_name, message_to_send, False, message_type, True)
+
+                print(f'RoomID: {room_id}: {create_message}')
+                from_user_info, to_user_info = to_user_info, from_user_info
+        else:
+           print(f'RoomID {room_id}: {register_message}') 
+        
+    else:
+        print(f'RoomID {room_id}: {message_room}')
+
+
+
+
 if __name__ == "__main__":
-    # comment out the method u want to run or run all 3 if you want to
+    print("--------------------------------------------------------------------------------------")
+    print(sys.argv)
     
-    # populate_rooms(rooms_api)
+    # supply 2 arguements in this order
+    # host_name: String => Ex: Jason, Abdul, etc
+    # host_type: String => Ex: ASL, STT
+    # Ex execution: populate_db.py Jason STT
+    #             : populate_db.py Will ASL
+    if len(sys.argv) > 2: 
```

Comment:
`sys.argv` works but in the future I would recommend using `argparse` instead. It's much cleaner and easier to use